### ISSUE TITLE
Plots

### ISSUE THREAD

**Issue by MickScandalous** [ForgeEssentials/ForgeEssentials#2055]
Im having a bug/issue with plots. When a player creates the plots it will say its all set up and it will work fine for a bit. Then after a restart or if they log off and come back it is like the plot doesnt exist. The plot perms are still there in the perms.json but as for the plot itself it is like it doesn exist in the world and other players can make plots there if they want. Also When I leave a plot it says im entering another plot that is no where neat the plot im exiting.

**Comment by spacebuilder2020** [comment]
Are you still having this issue?

**Comment by MickScandalous** [comment]
Yes I have just been using /zone and creating seperate zone for people

**Comment by securitypedant** [comment]
I have the same problem. I am using forgeessentials-1.7.10-1.4.4.1177-client and -server. I create a new plot using the following steps.

1. Define an area.
2. //expand 50 50 up
3. /plot define
4. /plot set name XXX
5. /plot set price 1

Then as another user, I stand in the plot and do...

/plot buy
/yes

And it states I now own the plot. Now if in the same logged in session I do a /plot list, it will show the plot. However, upon logging out and logging back and and running /plot list, the plot is no longer listed. However, it seems that the plot is actually there. The user that owns the plot has permissions to build and replace blocks. But others don't. So it just seems that the plot list commands are not working for some reason.

**Comment by olee** [comment]
Looks like the plot was not recognized as plot any more.
As plots are just areas with some special settings, it's not surprising that the permissions still work.
This would need some investigation....

**Comment by securitypedant** [comment]
I just built a fresh Forge 1.7.10-10.13.4.1614-1.7.10 server and installed only forgeessentials-1.7.10-1.4.4.1190-server and logged in with a client that only had forgeessentials-1.7.10-1.4.4.1190-client.

I created two plots, wandered around a bit. Logged out, and back in. The plots seemed to work perfectly fine.

I'm running a server with 77 mods and forgeessentials-1.7.10-1.4.4.1177-client. I'm going to test an upgrade to forgeessentials-1.7.10-1.4.4.1190-client and see if this bug goes away. If not, I suspect there is a problem with one of the mods I am running.

**Comment by MickScandalous** [comment]
yes and If you can find out what Mod that would be great! I am having a similar issue to you Securitypendant

**Comment by securitypedant** [comment]
I just tried 1.7.10-1.4.4.1190 on my server and the bug still resides. Problem is, I have 77 mods. I wonder which one is causing the problem. Mick, here is my list of mods. Let's at least see which we both have :D 

1.7.10-MB_Battlegear2-Bullseye-1.0.8.3.jar
adventurebackpack-1.7.10-0.8c.jar
AOBD-2.9.1.jar
Armourers-Workshop-1.7.10-0.39.2.jar
Baubles-1.7.10-1.0.1.10.jar
BiblioCraft[v1.11.4][MC1.7.10].jar
BigReactors-0.4.2A2.jar
BiomesOPlenty-1.7.10-2.1.0.1889-universal.jar
BloodMagic-1.7.10-1.3.3-17.jar
buildcraft-7.1.16.jar
buildcraft-compat-7.1.5.jar
Carpenter's Blocks v3.3.7 - MC 1.7.10.jar
cfm-3.4.8-mc1.7.10.jar
ChickenChunks-1.7.10-1.3.4.19-universal.jar
CodeChickenCore-1.7.10-1.0.7.47-universal.jar
CoFHCore-[1.7.10]3.1.2-325.jar
ComputerCraft1.75.jar
Decocraft-2.3.2_1.7.10.jar
defaultkeys-mc1.7.10-1.0.28.jar
DoggyStyle-1.1.3.99.jar
DrZharks MoCreatures Mod v6.3.1 (1).zip
EquivalentExchange3-1.7.10-0.3.0.519.jar
extrautilities-1.2.12.jar
FoodPlus-1.7.2-3.2rS.jar
forestry_1.7.10-4.2.12.60.jar
forgeessentials-1.7.10-1.4.4.1177-client.jar
Galacticraft-Planets-1.7-3.0.12.460.jar
GalacticraftCore-1.7-3.0.12.460.jar
Hats-4.0.1.jar
iChunUtil-4.1.2.jar
inventorypets-1.7.10-1.4.3-universal.jar
ironchest-1.7.10-6.0.62.742-universal.jar
journeymap-1.7.10-5.1.4p1-unlimited.jar
malisiscore-1.7.10-0.14.3.jar
malisisdoors-1.7.10-1.13.2.jar
Mantle-1.7.10-0.3.2b.jar
MCA-1.7.10-5.1.2-universal.jar
MicdoodleCore-1.7-3.0.12.460.jar
ModsList.txt
ModularPowersuits-0.11.0.300.jar
Morph-Beta-0.9.2.jar
NetherOres-[1.7.10]2.3.1-22.jar
NotEnoughItems-1.7.10-1.0.5.118-universal.jar
Numina-0.4.0.131.jar
Oceancraft-1.4.1.jar
OptiFine_1.7.10_HD_U_D4.jar
orespawn-1.7.10-20.3.zip
Pam's HarvestCraft 1.7.10Lb.jar
RadixCore-1.7.10-2.1.1-universal.jar
Railcraft_1.7.10-9.10.0.0.jar
resources
Ruins-1.7.10.jar
secretroomsmod-1.7.10-4.7.1.406.jar
SkinPort-1.7.10-v8c.jar
SolarFlux-1.7.10-0.8b.jar
TConstruct-1.7.10-1.8.8.jar
ThermalExpansion-[1.7.10]4.1.2-240.jar
ThermalFoundation-[1.7.10]1.2.3-112.jar
UndergroundBiomesConstructs-1.7.2-0.7.jar
UpdateCheckerMod-1.7.10.jar
Waila-1.5.10_1.7.10.jar
Wawla-1.3.3-1.7.10.jar
youtubers-1.8.0-MC1.7.10.jar
[1-7-10]_Lucky_Block_v5-1-0.jar
[1.7.10] SecurityCraft v1.8.1.jar
[1.7.10]Better_Furnaces_V4.3.jar
[1.7.10]MoVillages-1.4.1.jar
[1.7.10]Treecapitator-universal-2.0.4.jar

**Comment by securitypedant** [comment]
Mick, if you can list your mods. I will then build a simple test server with whatever mods we both share. If I can then re-create the bug, i'll make some educated guesses on what mod is causing the problem and remove it, bit by bit I might be able to figure out the offensive mod.

I wish I had the time to actually put together a dev environment. I could probably fix this myself.

**Comment by MickScandalous** [comment]
Okay I will get a list o all the mods we share here! Gie a ew to get it together, Your are the best! This is what I hae been waiting or

**Comment by MickScandalous** [comment]
My list is of Ones that you also have even if its a different update, it reduced it down to 30 or so. My keyboard must need cleaned cause that spelling sucked in my post before, sorry about that.

BiblioCraft[v1.11.4][MC1.7.10]
biomesoplenty-1.7.10-2.1.0
buildcraft-7.1.16
buildcraft-compat-7.1.5
Carpenter's Blocks v3.3.8_dev_r8 - MC 1.7.10
Baubles-1.7.10-1.0.1.10
CodeChickenCore-1.7.10-1.0.7.47-universal
CoFHCore-[1.7.10]3.1.2-325
Decocraft-v1.12b_1.7.10
DrZharks MoCreatures Mod v6.3.1
extrautilities-1.2.12 (1)
FoodPlus-1.7.2-3.2rS
forgeessentials-1.7.10-1.4.4.1164-server
Galacticraft-Planets-1.7-3.0.12.375
GalacticraftCore-1.7-3.0.12.375
Hats-4.0.1
iChunUtil-4.2.2
ironchest-1.7.10-6.0.62.742-universal
journeymap-1.7.10-5.1.4p1-unlimited
malisiscore-1.7.10-0.14.3
malisisdoors-1.7.10-1.13.2
Mantle-1.7.10-0.3.2
MicdoodleCore-1.7-3.0.12.375
Pam's HarvestCraft 1.7.10k
secretroomsmod-1.7.10-4.7.1.413
TConstruct-1.7.10-1.8.8
ThermalExpansion-[1.7.10]4.1.2-240
ThermalFoundation-[1.7.10]1.2.3-112
Waila-1.5.10_1.7.10
[1.7.10] SecurityCraft v1.8.1

**Comment by MickScandalous** [comment]
My first guess would be Security Craft, Id try that one first. Other than that its up in the air for me. Unfortunately I think you would be better at testing because my computer is not really made for that

**Comment by securitypedant** [comment]
I started work on this tonight. I got the following mods installed, progressively. I created a load of plots, and the /plot list command worked, apart from one odd issue. At one point, the list stopped listing a certain plot. I knew it existed.

Also, when I first started creating plots, walking in and out of them didn't report their name. I.e. "You are entering plot XXX".

But then after adding another mod and doing more testing, that functionality started to work. The plot thickens... (yes, pun 100% intended)

**Comment by Zethalion** [comment]
As we moved to 1.12.2, is this still an issue? Closing this issue in 7 days if no response @MickScandalous

**Comment by MickScandalous** [comment]
You tell me, you have had over 2 yrs to figure it out lol
